src/components/ListBox/ListBox.js

```diff
@@ -56,7 +57,7 @@ const ListBox = ({
         {...rest}
         role="listbox"
         aria-label={ariaLabel}
-        tabIndex="0"
+        tabIndex={innerTabIndex || 0}
```

Comment:
Just catching back up with this now, is there a specific reason why this is `innerTabIndex` as a prop versus supporting `tabIndex`?

Reply:
Didn't want to break other components that utilized listBox and expected the listBox being focusable, so provided a prop that could override that behaviour.

Reply:
@yacineMTB could you help me better understand the situation you're describing? I think I get what you're saying, but just want to confirm 😄

Reply:
Other components consume the listBox - and they may rely on the component being focusable for a11y purposes. 

In the case of the filterable multi select, we don't require the listbox to be focusable. I am led to believe this as the list box is explicitly set to be focusable. To avoid breaking other component's a11y, I added an override property that can override the tab index.